src/scripts/demo.js

```diff
@@ -143,7 +143,13 @@ const MOCK_COMFY = (function () {
 			if (channelNames.length) {
 				broadcaster.user = channelNames[0];
 			}
-			setTimeout(_generateNextMessage, 500);
+			if (window.CONFIG.DEMO === 'static') {
+				for (let i = 0; i < window.CONFIG.showLatestMessages; i++) {
```

Comment:
used the `showLatestMessages` query param option so the `DEMO=static` page respects that count.
That way "http://localhost:8080/c/test?theme=advent-of-code&DEMO=static&showLatestMessages=2" shows the advent-of-code theme with only the 2 random messages shown.
I figure this would be useful to develop when the messagecount needs to be exact (already seen some `nth` selectors for themes)

Reply:
I like this idea a lot!

Ideally, we'll retire the `nth` selectors in the themes now that `showLatestMessages` exists - but I've been handling that with staggered releases.